OrbitCore/SamplingProfiler.cpp

```diff
@@ -58,17 +58,15 @@ std::multimap<int, CallstackID> SamplingProfiler::GetCallstacksFromAddress(
 }
 
 //-----------------------------------------------------------------------------
-void SamplingProfiler::AddCallStack(CallStack& a_CallStack) {
-  CallstackID hash = a_CallStack.Hash();
+void SamplingProfiler::AddCallStack(CallstackEvent& callstack_event) {
```

Comment:
This seems now very similar to `SamplingProfiler::AddHashedCallstack`, with the only difference being the behavior in case of `!HasCallStack(...)`. Are they both necessary or can one of the two be removed?

Reply:
I probably want to refactor SamplingProfiler in one of the upcoming PRs, will keep it in mind, thanks!

Reply:
Done in #865